includes/drush/custom/toolkit.drush.inc

```diff
@@ -69,9 +69,8 @@ function drush_toolkit_check_modules_authorized_security($project_id = NULL) {
     // Check update module status.
     $updateStatus = system_get_info('module', 'update');
     // Enable 'Update Manager' module if it's disabled.
-    if (empty($updateStatus)) {
-      $updateModule = array('update');
-      module_enable($updateModule, FALSE);
+    if (!module_exists('update') && empty($updateStatus)) {
```

Comment:
I don't think we need this anymore `empty($updateStatus)`.

Reply:
Yes @zarabatana , you are correct. I have removed it.